### PR TITLE
fixes #331 - Handled case when no app matched search

### LINKED ISSUE #331: Handle case when no app matches search on front page
Proper message should be displayed if no app matches search term.

### PR DESCRIPTION
**Pull request by djmgit**
Fixes issue #331, proper message is displayed if search matches
no app.

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by djmgit** [comment]
Test link: http://loklakapps.surge.sh/

@kavithaenair @Achint08 @SKrPl @vibhcool open for review

**Comment by djmgit** [comment]
@kavithaenair resolved it, also removed codacy issues, please re-review.

File: css/dashboard.css [hunk 1]
```diff
@@ -407,6 +407,15 @@ section{
   margin-top: -50px;
 }
 
+.not-found {
+  margin-top: 500px;
+  font-size: 32px;
+  position: absolute;
+  top: -30%;
```
Comment (Achint08) [inline css/dashboard.css]: Please improve CSS a bit.

**Comment by vibhcool** [comment]
@djmgit , please see if some more issues can be resolved

**Comment by djmgit** [comment]
@vibhcool @Achint08 resolved some of the codacy issues and removed unnecessary margin rule, please re-review

**Pull request merged**